war_or_peace_runner.rb

```diff
@@ -0,0 +1,79 @@
+class Start
```

Comment:
Our runner shouldn't be defining a class. If we want a class to hold these behaviors, it should go in the lib directory. Our runner shouldn't have very much code in it. Probably just a couple lines to actually start the game.